internal/controlplane/handlers_profile.go

```diff
@@ -240,17 +231,15 @@ func getRuleEvalEntityInfo(
 	entityType *db.NullEntities,
 	selector *uuid.NullUUID,
 	rs db.ListRuleEvaluationsByProfileIdRow,
-	providerName string,
 ) map[string]string {
 	l := zerolog.Ctx(ctx)
-	entityInfo := map[string]string{
-		"provider": providerName,
-	}
+	entityInfo := map[string]string{}
```

Comment:
I'm assuming this won't break existing clients?

Reply:
The provider still gets added later in the flow, and we use the provider from the repo